spec/sources/github_spec.rb

```diff
@@ -33,6 +33,13 @@ def fixture(file)
     expect(g.api_url).to eql("https://api.github.com/repos/artsy/eigen/pulls/800")
   end
 
+  it 'gets the right url from buildkite' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.